engine/consensus/sealing/core.go

```diff
@@ -29,28 +29,20 @@ import (
 	"github.com/onflow/flow-go/utils/logging"
 )
 
-// DefaultRequiredApprovalsForSealConstruction is the default number of approvals required to construct a candidate seal
-// for subsequent inclusion in block.
-// when set to 1, it requires at least 1 approval to build a seal
-// when set to 0, it can build seal without any approval
-const DefaultRequiredApprovalsForSealConstruction = 1
-
 // DefaultEmergencySealingActive is a flag which indicates when emergency sealing is active, this is a temporary measure
 // to make fire fighting easier while seal & verification is under development.
 const DefaultEmergencySealingActive = false
 
 // Config is a structure of values that configure behavior of sealing engine
 type Config struct {
-	EmergencySealingActive               bool   // flag which indicates if emergency sealing is active or not. NOTE: this is temporary while sealing & verification is under development
-	RequiredApprovalsForSealConstruction uint   // min number of approvals required for constructing a candidate seal
-	ApprovalRequestsThreshold            uint64 // threshold for re-requesting approvals: min height difference between the latest finalized block and the block incorporating a result
+	EmergencySealingActive    bool   // flag which indicates if emergency sealing is active or not. NOTE: this is temporary while sealing & verification is under development
```

Comment:
Any reason not to keep the updatable field with the rest of the config?
```suggestion
	EmergencySealingActive    bool   // flag which indicates if emergency sealing is active or not. NOTE: this is temporary while sealing & verification is under development
	RequiredApprovalsForSealConstruction updatable.UintGetter
```

Reply:
> The updatable field is managed by a stateful instance, which needs to be referenced as pointer, where as Config is passed as value.

I don't understand why this would prevent us from including the `SealingConfigsGetter` with the other config:
```go
type Config struct {
    ...
    sealingConfig SealingConfigsGetter
}
```
Config can be passed by value, `sealingConfig` is a pointer field, copies of Config point to the same instance of `sealingConfig`.

Alex's suggestion of putting all the config in the `SealingConfigsGetter` seems like another good approach.